app/views/common/os_selection/_pxe_loader.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= fields_for item do |f| -%>
+  <% if @operatingsystem.available_loaders(item.architecture).any? -%>
+    <% selected_attr = f.object.new_record? && item.hostgroup.present? && item.hostgroup.pxe_loader.nil? ? { :selected => @operatingsystem.preferred_loader(item.architecture) } : {} -%>
```

Comment:
I think, a better approach is to do this during host initialization. E.g. add something like this to `#initialize´ in `Host::Base`:

```
self.pxe_loader ||= 'pxelinux.0'
```

Does `item.hostgroup.pxe_loader` contain the inherited values as well?